.pre-commit-config.yaml

```
@@ -1,12 +1,12 @@
 repos:
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.15.0
+    rev: v3.15.2
     hooks:
       - id: pyupgrade
         args: [--py38-plus]
 
   - repo: https://github.com/psf/black
```

Has faster mypyc-compiled wheels:
```suggestion
  - repo: https://github.com/psf/black-pre-commit-mirror
```